scripts/convertpsl.py

```diff
@@ -27,7 +27,7 @@ def convert(psl_text):
     with open(sys.argv[1], 'r+') as f:
         psl = convert(f)
         f.seek(0)
-        f.write('var publicSuffixes = %s;' % (
-            json.dumps(psl, sort_keys=True, indent=4, separators=(',', ': '))
+        f.write('window.publicSuffixes = %s;' % (
```

Comment:
I can't comment on that line, but should we update basedomain.js to access publicSuffixes as `window.publicSuffixes` and match this?

Reply:
Maybe ... I moved `basedomain.js` to `vendor/` in this PR because I thought of it as a third-party library we weren't going to modify, but maybe I should move it back and fix all the lint complaints.

Reply:
Done and done.